modules/restoring-red-hat-quay.adoc

```diff
@@ -0,0 +1,237 @@
+====
+If you receive the error `Error from server (AlreadyExists): error when creating "./config-bundle.yaml": secrets "config-bundle-secret" already exists`, you must delete your exist resource with `$ oc delete Secret config-bundle-secret -n <quay-namespace>` and recreate it with `$ oc create -f ./config-bundle.yaml`.
+====
+
+. Restore the QuayRegistry custom resource:
++
+[source,terminal]
+----
+$ oc create -f ./quay-registry.yaml
+----
+
+. Scale down the Quay component and edit the QuayRegistry custom resource by unmanaging the `horizontalpodautoscaler` component:
++
+[source,terminal]
+----
+$ oc patch QuayRegistry <quay-registry-name> -n  <quay-registry-namespace>  -p '{"spec": {"components": [{"kind": "horizontalpodautoscaler", "managed": false }]}}' --type='merge'
+----
++
+Example YAML:
++
+[source,yaml]
+----
+  apiVersion: quay.redhat.com/v1
+  kind: QuayRegistry
+  metadata:
+    name: registry
+    namespace: ns
+  spec:
+    components:
+      …
+      - kind: horizontalpodautoscaler
+        managed: false
+      - kind: quay
+        managed: true
+        overrides:
+          replicas: 0
+      …
+----
++
+[NOTE]
+====
+If you are using a version older than 3.7.0, then you must manually scale down the Operator and manually scale down the Quay deployment. For example, you can run `$ oc scale --replicas=0 deployment quay-operator.v3.6.2 -n openshift-operators` to scale down the Operator, and `$ oc scale --replicas=0 deployment QUAY_MAIN_APP_DEPLOYMENT` to scale down the main application deployment.
+====
+
+. Identify your Quay database pod:
++
+[source,terminal]
+----
+$ oc get pod -l quay-component=postgres -n <quay-namespace> -o jsonpath='{.items[0].metadata.name}'
+----
++
+Example output:
++
+----
+example-registry-quay-database-66969cd859-n2ssm        1/1     Running            0          6d1h
+----
+
+. Upload the backup by copying it from the local environment and into the pod:
++
+----
+$ oc cp ./backup.sql -n <quay-namespace> registry-quay-database-66969cd859-n2ssm:/tmp/backup.sql
+----
+
+. Open a remote terminal to the database:
++
+[source,terminal]
+----
+$ oc rsh -n <quay-namespace> registry-quay-database-66969cd859-n2ssm
+----
+
+. Enter psql:
++
+[source,terminal]
+----
+bash-4.4$ psql
+----
+
+. You can list the database by running the following command:
++
+----
+postgres=# \l
+----
++
+Example output:
++
+[source,terminal]
+                                                  List of databases
+           Name            |           Owner            | Encoding |  Collate   |   Ctype    |   Access privileges
+----------------------------+----------------------------+----------+------------+------------+-----------------------
+postgres                   | postgres                   | UTF8     | en_US.utf8 | en_US.utf8 |
+quayregistry-quay-database | quayregistry-quay-database | UTF8     | en_US.utf8 | en_US.utf8 |
+
+
+. Drop the database:
++
+----
+postgres=# DROP DATABASE "registry-quay-database";
+----
```

Comment:
For consistency with line 118, here the database name should be `quayregistry-quay-database`.

----
postgres=# DROP DATABASE "quayregistry-quay-database";
----